ui/pages/unlock-page/index.scss

```diff
@@ -9,7 +9,6 @@
   color: var(--color-text-default);
 
   &__container {
-    background: var(--color-background-default);
     display: flex;
     align-self: stretch;
     justify-content: center;
```

Comment:
Fixes overlapping div on border of container

<img width="436" alt="Screen Shot 2022-04-18 at 5 59 19 PM" src="https://user-images.githubusercontent.com/8112138/164033328-ccbfd612-c266-418e-a8da-3f7bc81b4595.png">

Reply:
This might need a revisit because it removes the background of the default screen
<img width="1091" alt="Screen Shot 2022-04-19 at 8 18 43 AM" src="https://user-images.githubusercontent.com/8112138/164038000-20e10f08-8c25-4662-803a-c4aed5a67e58.png">

Reply:
Just kidding the added padding allows for this fix :D 

<img width="625" alt="Screen Shot 2022-04-20 at 1 54 37 PM" src="https://user-images.githubusercontent.com/8112138/164321214-2818afe7-d109-463f-997f-f19596e3d271.png">
<img width="1385" alt="Screen Shot 2022-04-20 at 1 54 50 PM" src="https://user-images.githubusercontent.com/8112138/164321215-24c7a1b3-30e7-4fc3-b45a-1e8d9649b8ef.png">